tasks/task452_emea_bg_el_classification.json

```diff
@@ -0,0 +1,1061 @@
+{
+    "Contributors": [
+        "Vijay Kumawat"
+    ],
+    "Source": [
+        "emea bg-el (dataset link- https://huggingface.co/datasets/emea)"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Input_language": "Bulgarian",
+    "Output_language": "Greek",
+    "instruction_language": "English",
```

Comment:
Do you need these three fields @swarooprm ?

Reply:
yes fine